### PR TITLE
fix(loader): Fix the loader which was not displayed

### LINKED ISSUE #38: Front - Android - Loader on login page
When you enter your email and password, after the click of the button "Se connecter", the app loader don't display
Problematic code : LoginPage.kt line 103

### PR DESCRIPTION
**Pull request by NyraSama**
# Description
What are changes related to ?

Make the loader visible when we try to connect to the app

# Tests
How has it been tested ?

- [X] Manually
- [ ] Automated tests
- [ ] QA
- [ ] Other

### CODE REVIEW

File: app/src/main/java/com/xpeho/xpeapp/ui/presentation/page/LoginPage.kt [hunk 1]
```diff
@@ -100,11 +102,10 @@ fun LoginPage(
                     }
                 }
                 when (vm.wordpressState) {
-                    // A comprendre pourquoi le apploader ne s'affiche pas
                     is WordpressUiState.LOADING -> AlertDialog(
                         onDismissRequest = { /*TODO*/ }
                     ) {
-                        AppLoader()
+                        //AppLoader()
```
Comment (mmarchal) [inline app/src/main/java/com/xpeho/xpeapp/ui/presentation/page/LoginPage.kt]: If it's not trigger, we can delete this part of code

**Comment by NyraSama** [comment]
# Preview

https://github.com/XPEHO/XpeApp/assets/83142634/88f88500-4829-4fbb-95fe-38ff50c16111

File: app/src/main/java/com/xpeho/xpeapp/ui/presentation/page/LoginPage.kt (hunk 1 above)
Comment (NyraSama) [reply app/src/main/java/com/xpeho/xpeapp/ui/presentation/page/LoginPage.kt]: I keep it because of the TODO, maybe you wanted to add something here ?

Comment (mmarchal) [reply app/src/main/java/com/xpeho/xpeapp/ui/presentation/page/LoginPage.kt]: No, the comment is to find why is doesn't work but you have develop another way so it's good for me

**Review by mmarchal — APPROVED** [review]

**Pull request merged**